superset-frontend/src/dashboard/components/nativeFilters/FilterCard/FilterCard.test.tsx

```diff
@@ -204,6 +205,13 @@ jest.mock('@superset-ui/core', () => ({
   }),
 }));
 
+jest.mock(
```

Comment:
I thought this is what you had to add to avoid `act` warnings when running tests – without it all of the tests output a bunch of `act` icon errors.  @michael-s-molina I know you worked on this, is this still the best case?

Reply:
Given that our icons load asynchronously, you can either mock the icons as you did or wait for them to render. We generally mock them when they're not the focus of the tests.